templates/tracker.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Simple chat bot</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+        crossorigin="anonymous">
+    <link type="text/css" rel="stylesheet" href="../static/css/style.css">
```

Comment:
the `href` part should be `/static/css/pachimari.css`. You don't need the double dot here. You should only use this directory notation if this html file is served statically. In this case it is being served statically by gin.

Reply:
@lamngue